### ISSUE TITLE
EmptyLineSeparator: false positive "',' should be separated from previous line" on multi-variable field declaration

### ISSUE THREAD

**Issue by vivek-0509** [checkstyle/checkstyle#20330]
I have read check documentation: https://checkstyle.org/checks/whitespace/emptylineseparator.html
I have downloaded the latest checkstyle from: https://checkstyle.org/cmdline.html#Download_and_Run
I have executed the cli and showed it below, as cli describes the problem better than 1,000 words

```bash
vivek@Viveks-MacBook-Air checkstyle % cat > config.xml <<'EOF'
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
    "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
    "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">
  <module name="TreeWalker">
    <module name="EmptyLineSeparator"/>
  </module>
</module>
EOF

vivek@Viveks-MacBook-Air checkstyle % cat > Example.java <<'EOF'
class Example {
    int a = 1, b = 2;
}
EOF

vivek@Viveks-MacBook-Air checkstyle % RUN_LOCALE="-Duser.language=en -Duser.country=US"
java $RUN_LOCALE -jar target/checkstyle-13.6.0-SNAPSHOT-all.jar -c config.xml Example.java

Starting audit...
[ERROR] /Users/vivek/checkstyle/Example.java:2:14: ',' should be separated from previous line. [EmptyLineSeparator]
Audit done.
Checkstyle ends with 1 errors.
```
---

**Describe what you expect in detail.**

A single multi-variable field declaration such as `int a = 1, b = 2;` should not produce an EmptyLineSeparator violation on the comma.
`a` and `b` are two `VARIABLE_DEF` nodes joined by a comma within one declaration. With the default `allowNoEmptyLineBetweenFields = false`, the check treats them as two fields that must be separated by an empty line and flags the comma (`isViolatingEmptyLineBetweenFieldsPolicy` special-cases the comma; setting `allowNoEmptyLineBetweenFields = true` suppresses it).  The requirement cannot be satisfied, you cannot put a blank line inside a single declaration.

Expected: a multi-variable declaration on one line should not be flagged on the comma by default.

**Comment by surajgojanur** [comment]
i'm working on this
